Products/CMFPlone/Portal.py

```diff
@@ -222,8 +222,7 @@ def isEffective(self, date):
         return 1
 
 
-# Remove the IContentish interface so we don't listen to events that won't
-# apply to the site root, ie handleUidAnnotationEvent
-classImplementsOnly(PloneSite, implementedBy(PloneSite) - IContentish)
+# Add the IContentish interface back to the PloneSite class
+classImplementsOnly(PloneSite, implementedBy(PloneSite) + IContentish)
```

Comment:
You can drop the whole line. `IContentish` was already part of `PloneSite` - I just removed it with `classImplementsOnly`.

If we succesfully no longer remove `IContentish`, we should start seeing test failures involving `handleUidAnnotationEvent` in the coredev tests.